.github/workflows/ci.yml

```
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  push:
+  push:
```

actionlint (1.7.8)

key "push" is duplicated in "on" section. previously defined at line:4,col:3

(syntax-check)